blocks/basic/include/gnuradio-4.0/basic/And.hpp

```diff
@@ -0,0 +1,29 @@
+#ifndef AND_HPP
+#define AND_HPP
+
+#include <gnuradio-4.0/Block.hpp>
+#include <gnuradio-4.0/BlockRegistry.hpp>
+
+namespace gr::basic {
+
+GR_REGISTER_BLOCK(gr::basic::And, [uint8_t, int16_t, int32_t])
+
+template<typename T>
+requires std::is_arithmetic_v<T>
```

Comment:
`std::is_arithmetic_v<T>` also includes. This is probably not what you intended. 

Maybe the concept [std::integral](https://en.cppreference.com/w/cpp/concepts/integral) might be a better choice, i.e. `template<std::integral>`.